help_to_heat/templates/frontdoor/base.html

```diff
@@ -29,7 +29,7 @@
 </head>
 
 <body class="govuk-template__body" id="i-dot-ai">
-  {% if request.COOKIES['cookiesAccepted'] != "True"%}
+  {% if request.COOKIES['cookiesAccepted'] != "True" and request.COOKIES['cookiesAccepted'] != "False"%}
```

Comment:
We currently don't have tests for cookie banner. Do we want any?

Reply:
Yeah, we should get some in. The non-JS version should be relatively easy to test.

Ideally wanna check all of:
- if you accept, you get acceptance cookie saying yes
- if you reject, you get acceptance cookie saying no
- if you have acceptance cookie yes then you get other cookies
- if you have acceptance cookie no then you don't get other cookies

Reply:
Update: we're not adding any new tests for now